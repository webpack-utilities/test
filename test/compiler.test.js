const { webpack } = require('../src/index.js')

describe('Compiler', () => {
  test('Default', async () => {
    const config = {}

    const stats = await webpack('fixture.js', config)
    const { source } = stats.toJson().modules[0]
    
    expect(source).toMatchSnapshot()
  })
})