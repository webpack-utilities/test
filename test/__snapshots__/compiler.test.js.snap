// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compiler Default 1`] = `"export default 42"`;
